packages/dashboard-message-bus-client/package.json

```diff
@@ -42,12 +42,13 @@
     "axios": "^0.24.0",
     "debug": "^4.3.1",
     "delay": "^5.0.0",
-    "isomorphic-ws": "^4.0.1"
+    "isomorphic-ws": "^4.0.1",
+    "tiny-typed-emitter": "^2.1.0",
+    "ws": "^7.2.0"
```

Comment:
Confused why this dependency was added; I can't find it being used anywhere in the package, and the new depcheck script doesn't complain here either.  What's the reason for adding this?

Reply:
[Seems to be a peer dep of `isomorphic-ws`](https://github.com/heineiuo/isomorphic-ws/blob/master/package.json#L26)

Reply:
Oh, huh!  Are there other places we have to deal with peer deps like this?  Could there be more of these that are missing?  (...not that those need to go in this same PR, mind you).

Reply:
So I caught this because one of the import was breaking without it, can't recall which.

I think `isomorphic-ws` across the monorepo now has `ws` peer dep, so that's fine. But I'm not sure on how to find other peer dep issues like this, @cds-amal might know? Does [`dependency-check`](https://github.com/dependency-check-team/dependency-check) have a suitable flag for this?

Related: [NPM v7 automatically installs peer deps](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#peer-dependencies)

Reply:
It doesn't seem like `dependency-check` handles this, unfortunately.  Troublesome.

Reply:
I'm thinking we can be more confident if we utilize `dependency-check` along with the `verdaccio` testing outlined in the PR's `How to test` section.